### ISSUE TITLE
Would you please provide an example code?

### ISSUE THREAD

**Issue by HaFred** [Luttik/keras_svm#1]
Hi Luttik,
I am also trying to implement kernel function in the first hidden layer using Keras. I found your repo but I don't know how to use it specifically. Would you please provide an example? A simple SVM example using your package would be fine! 
Thank you very much!

**Comment by Luttik** [comment]
Hi @HaFred,

This is not the purpose of this package, I just replaced the softmax layer with a linear svm. You can however still do this. Keras allows you to define your own layers [(documentation)](https://keras.io/layers/writing-your-own-keras-layers/). It seems that it is just a question of inserting the mathematical definition of your kernel.
